src/services/auth.ts

```diff
@@ -41,6 +41,12 @@ firebase.initializeApp({
 
 const googleProvider = new firebase.auth.GoogleAuthProvider()
 
+const bearer = (idToken: string) => ({
```

Comment:
I would have called it `createHeaders` or something like this.

Reply:
Yes create createHeaders makes more sense